src/Polymorph-Widgets/UITheme.class.st

```diff
@@ -754,12 +754,12 @@ UITheme >> chooseColorIn: aThemedMorph title: aString color: aColor for: aBlock
 UITheme >> chooseDirectoryIn: aThemedMorph title: title path: path [
 	"Answer the result of a file dialog with the given title, choosing directories only."
 
-	| dialog |
-	dialog := (FileDialogWindow newWithTheme: aThemedMorph theme)
-		title: title;
-		answerDirectory.
-	path ifNotNil: [ dialog selectPath: path ].
-	^ dialog openModal answer
+	^ StOpenDirectoryDialog new
+		defaultFolder: path;
+		title: (title ifNil: [ 'Choose Directory' translated ]);
```

Comment:
This does not work.
Title if provided is not displayed as the window is not yet open

Reply:
It looks buggy certainly. I don't know because I never use `translate`  :)
Anyway this could be soon deprecated, @jecisc ?

Reply:
translated should return a string translated in the local language if the translation project is loaded IIRC. In the worst case it should return the original string in any case.

Reply:
I think that some of the API should get deprecated but I'm not yet sure what will get deprecated exactly and right now I'm working on Moose

Reply:
The problem here is not the translation. Setting the title before the window is open has no effect.
You can try by yourself:
```smalltalk
StOpenDirectoryPresenter new
		title: 'test set tittle';
		openModal
```

Reply:
I've sent a PR in https://github.com/pharo-spec/NewTools/pull/716 to fix this

Reply:
Merged